### DISCUSSION TITLE
YouTube出手制裁「廣告攔截器」！ 用戶使用一律降速5秒

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1713] [General]
https://tw.news.yahoo.com/youtube%E5%87%BA%E6%89%8B%E5%88%B6%E8%A3%81-%E5%BB%A3%E5%91%8A%E6%94%94%E6%88%AA%E5%99%A8-%E7%94%A8%E6%88%B6%E4%BD%BF%E7%94%A8-%E5%BE%8B%E9%99%8D%E9%80%9F5%E7%A7%92-072341634.html

這真的超煩的拉，而且沒買yt premium 還要被"禁止廣告攔截"的廣告干擾，真的會中風

**Comment by 12Jesse24** [comment]
真的很煩。每次放著聽歌一首歌結束後都要等15到20秒，極度影響觀賞流暢性，結論所以你得買yt premium才能體會0廣告，而且我覺得這現象會越來越嚴重，可能過幾年廣告的秒數就是1分鐘起跳了XD

**Comment by lazywulf** [comment]
維持五秒我覺得還好，總比連塞兩個 15+ 廣告好多了。
不過擋 ad blocker 這檔事其實也蠻討厭的。廣告也不審，只想著靠那幾個爛廣告來煩人去買 premium，想得美呢。

我用 brave 我超棒 (x

**Comment by Ryanlolry** [comment]
現在感覺要做到正版網站零廣告感覺是不可能的，除非使用盜版網站，或使用付費的正版網站，畢竟網站的營運成本再升高，他必須開發更多賺錢的的手段來賺錢

**Comment by v0ouo** [comment]
我覺得買youtube premium應該是最好的解決方法了，而且買youtube premium也沒什麼不好啊，我自己用youtube music用的還挺開心的。而且還可以保留上次看的影片，手機還可以用小視窗看。不過建議跟人一起共組家庭買，可以少花一點錢

**Comment by WeiYen1103** [comment]
我覺得這個超級酷，因為前陣子YT開始擋掉外掛，所以後來有人開始開發讓廣告在1ms播完的方式，希望之後能開發出來XD，但其實我一直以來都沒有安裝過任何外掛程式，也不知道會不會有病毒QQ

**Comment by davidteng00000** [comment]
Youtube對廣告攔截採取降速5秒的處罰，確實令人不悅。儘管購買Premium可享零廣告，但這樣的限制可能會推動更多人尋找其他方式，對於網站和觀眾而言，這是個挑戰。但是我還是買premium了ww

**Comment by 1chooo** [comment]
推薦大家使用 Brave 瀏覽器，我自己用下來是直接擋廣告的（無須安裝 AD Block），是個開源的瀏覽器，核心也跟 Chrome 一樣，所以用習慣 Chrome 的人來說可以無痛切換

**Reply by 1chooo** [reply]
附上 GitHub Repo 和官網給大家

- GitHub Repo: https://github.com/brave/brave-browser
- Brave Official Site: https://brave.com/

**Comment by chuanmin0317** [comment]
我自己是有訂youtube premium，我覺得如果有人一起分擔月費的話並不算貴，畢竟現在的youtube廣告越來越長，而我使用youtube的時間也不算短，付一點點錢來減少看廣告浪費的時間我覺得蠻值得的

**Comment by woohsin** [comment]
之前都用ad blocker，結果聽說要制裁之後我就開始課金了，但是聽其他人說其實好像也不不會真的對你做什麼:)但是花錢了事有時候覺得滿方便的。

**Comment by woohsin** [comment]
我聽同學說瞬間按上一頁下一頁就可以跳過了耶：）

**Comment by YANRONG0123** [comment]
我目前都使用uBlock去阻擋YT上的廣告，真的沒有那個閒錢去使用premium，至於使用方法自行上網搜尋，然後原本的adblock記得要在YT上永久停用，不然沒記錯的話，載入會有問題。

**Comment by EPHMR0524** [comment]
使用者付費我覺得本身沒有問題，置入廣告我也沒意見，但是廣告插入的時間常常是在的影片播到一半的時候，這相當影響觀賞體驗，而且還會出現些不可跳過的廣告，比例還越來越多，只讓人覺得吃相難看，這邊推薦使用Brave瀏覽器，youtube就都不會有廣告了

**Comment by 110101501** [comment]
現在youtube的廣告時間不單是變長了，而且還多了一個廣告，導致每次要等廣告都很耗時間。
雖然我自己是有在使用ad block ，但有時候會被youtube攔截，而無法正常使用youtube，蠻困擾的，會讓使用者愈來愈不願意使用youtube。

**Comment by happy246810** [comment]
現在youtuber除了有越來越多，越來越長的廣告，最讓我討厭的是越來越多廣告的模式都變成不可略過，原本可以略過還好，而現在則是連略過都不行。因此我都用ad blocker，但我應該會換成ubloke，以防被抓。

**Comment by YOGA012** [comment]
我是用Brave瀏覽器，可以免費阻止大部分網站的第三方廣告，不需要安裝ad block就可以不受廣告干擾看YouTube ，真的很好用，而且身邊平常有在玩音樂的朋友說Brave瀏覽器的音質比其他瀏覽器好，不知道是不是真的，反正我是聽不出來，有興趣的朋友們可以安裝來試試看。

**Comment by jeff110303549** [comment]
這確實是一個令人困擾的情況。廣告攔截器通常是用戶為了避免不必要的廣告干擾而使用的工具，但現在一些平台可能會開始對這些工具進行限制或者提醒用戶停用它們。這可能會對一些用戶造成不便，特別是那些沒有購買付費版本的用戶。
對於這樣的情況，我們可以向平台提供反饋，讓他們知道用戶的關注點，希望他們能夠找到一個更好的解決方案，既能保證廣告收入，又能確保用戶的良好體驗。
